docs/reference/search-application/apis/put-search-application.asciidoc

```diff
@@ -85,12 +99,36 @@ PUT _application/search_application/my-app?create
         "query_string": "*",
         "default_field": "*"
       }
+    },
+    "dictionary": {
+      "properties": {
+        "query_string": {
+          "type": "string"
+        },
+        "default_field": {
+          "type": "string",
+          "enum": [
+            "title",
+            "description"
+          ]
+        },
+        "additionalProperties": false
+      },
+      "required": [
+        "query_string"
+      ]
     }
   }
 }
 ----
 // TEST[skip:TBD]
 
+Given the value of the `dictionary` parameter, the <<search-application-search, search application search>> API will perform the following parameter validation:
+
+* It accepts only the `query_string` and `default_field` parameters
+* It verifies that `query_string` and `default_field` are both strings
+* It accepts `default_field` only if it takes the values `title` or `description`
+
 The following example creates or updates an existing Search Application called `my-app`:
```

Comment:
For this example is it worth including a sample response if input that violates the defined dictionary is sent in?